### PR TITLE
47: minor fixes

### LINKED ISSUE #47: minor fixes before CRAN submission
To do:

- [x] use `stats::integrate` to avoid note
- [x] add `--as-cran` to project file
- [x] put `survival` and `AHR` in `Suggests` because it is used in vignettes

### PR DESCRIPTION
**Pull request by danielinteractive**
closes #47
[skip vbump]

### CODE REVIEW

**Review by numbersman77 — APPROVED** [review]

File: vignettes/trialplanning.Rmd [hunk 1]
```diff
@@ -112,13 +111,13 @@ StudyforAvg <- getClinicalTrials(
   dropout = list(rate = 0.05, time = 12), accrual = list(param = "intensity", value = 25)
 )
 
-averageHR <- lapply(StudyforAvg, avergHR)
-MeanAverageHR <- mean(unlist(averageHR))
+averageHR <- sapply(StudyforAvg, avergHR)
+MeanAverageHR <- mean(averageHR)
 ```
 
+In our example, we get an average OS HR of `r round(MeanAverageHR, 3)`.
```
Comment (numbersman77) [inline vignettes/trialplanning.Rmd:118]: This is the average over all trials? For illustration would it not be enough to simply simulate one trial and compute the AHR?

**Review by numbersman77 — COMMENTED** [review]
Added one small comment

File: vignettes/trialplanning.Rmd (hunk 1 above)
Comment (danielinteractive) [reply vignettes/trialplanning.Rmd:118]: that is how it was before, yes. @niessl what do you think?

Comment (niessl) [reply vignettes/trialplanning.Rmd:118]: I took the average over several trials to get a more robust estimate

**Review by niessl — APPROVED** [review]

**Pull request merged**